src/app/book/add/add.component.css

```diff
@@ -0,0 +1,3 @@
+.bigger {
+  width: 350px;
+}
```

Comment:
You can do this using inline flex. Example: `fxFlex=" 1 1 350px"` . Although I'm not sure that a 350px baseline width is responsive.